modules/planning/common/message_process.cc

```diff
@@ -326,8 +314,9 @@ void MessageProcess::ProcessOfflineData(const std::string& record_file) {
   }
 }
 
-int MessageProcess::GetADCCurrentRoutingIndex() {
-  if (localizations_.empty()) return -1;
+bool MessageProcess::GetADCCurrentRoutingIndex(
+    int* road_index, double* road_s) {
+  if (localizations_.empty()) return false;
```

Comment:
Add `{
}`

Reply:
"Short conditional statements may be written on one line... "
the sample has no bracket. :)
https://google.github.io/styleguide/cppguide.html#Conditionals